ts/components/screens/BaseHeader.tsx

```diff
@@ -152,7 +153,9 @@ class BaseHeaderComponent extends React.PureComponent<Props, State> {
         return;
       }
       setAccessibilityFocus(
-        this.firstElementRef,
+        this.props.statusAppRef && this.props.statusAppRef?.current !== null
+          ? this.props.statusAppRef
+          : this.firstElementRef,
```

Comment:
Works fine on Android 🥳 

I think this fix can be called done once we an _ad-hoc_ test covering this specific issue: an assertion of `setAccessibilityFocus` arguments will be more than enough.